amplify/src/components/SignIn.js

```diff
@@ -1,21 +1,19 @@
 import React from 'react';
-import firebase, { provider } from '../firebase';
+import { Link } from 'react-router';
 
 const SignIn = () => {
```

Comment:
With a dumb component, you need to actually pass in the props that you mapped with redux. So this line would look something like:

```javascript
const SignIn = (props) => {
```